guide/sections/part2/global-services.adoc

```diff
@@ -202,9 +204,63 @@ wis2-gdc provides the functionality required for the Global Discovery Catalogue,
 
 wis2-gdc is managed as a free and open source project. Source code, issue tracking and discussions are hosted openly on GitHub: https://github.com/wmo-im/wis2-gdc.
 
-==== 2.7.6 Global Monitor
+==== 2.7.6 Global Replay
 
 ===== 2.7.6.1 Technical considerations
+
+* The Global Replay provides Global Services and data consumers with a mechanism to search, query and subscribe to notification messages and monitoring events of interest.
+* The Global Replay implements the OGC API – Features – Part 1: Core standardfootnote:[https://docs.ogc.org/is/17-069r4/17-069r4.html], adhering to the following conformance classes and their dependencies:
+** Core
+** GeoJSON
+* The Global Replay implements the OGC API – Processes - Part 1: Core standardfootnote:[https://docs.ogc.org/is/18-062r2/18-062r2.html], adhering to the following conformance classes and their dependencies:
+** Core
+** OGC Process Description
+** JSON
+* A Global Replay shall subscribe to a Global Broker instance, to the topics `+origin/a/wis2/#+`, `+cache/a/wis2/#+` and `+monitor/a/wis2/#+`.
+* The Global Replay will make notification messages available via the API collection identifier `wis2-notification-messages`.
+* The Global Replay will make monitoring events available via the API collection identifier `wis2-monitoring-events`.
+* The Global Replay will provide an API process to trigger the republication of previously sent messages, allowing users to subscribe to messages with various criteria (spatial and / or temporal extent, topic selection) via the Global Broker.
+** the API process request accepts the following inputs:
+*** `datetime`: a temporal range of messages to republish (example ``2025-04-06T17:21:26Z/2025-04-06T19:21:26Z``).
+*** `subscriber-id`: a unique identifier (UUID) defined by the user/client.
+*** `topic-id`: the topic on which to trigger message replication (centre-id required, allowing `+` and `#` for subsequent topic levels)
+** the resulting topic subscription in the API process response will be `+replay/a/wis2/<centre-id>/<callback>/<wth-topic>+`, where:
+*** ``callback`` is defined by the user/client.
+*** ``wth-topic`` is the full topic of the message as previously published to WIS2.
+** the API process can set temporal boundary limits which, when exceeded, result in an error message prohibiting the republishing of messages.
+*** the temporal boundary is 15 minutes.
```

Comment:
Wording of 230 and 231 is strange. 
Why not imposing directly the 15 minutes ? 

** the API process will set temporal boundary limits to 15 minutes. When exceeded, it will result in an error message prohibiting the republishing of messages.